types/material-ui/material-ui-tests.tsx

```diff
@@ -109,7 +109,13 @@ function handleTouchTap() {
   alert('onTouchTap triggered on the title component');
 }
 
-const styles = {
+type Stylesheet<T> = {
+  [P in keyof T]: React.CSSProperties
+};
+
+declare function createStylesheet<T>(args: T): Stylesheet<T>;
+
+const styles = createStylesheet({
```

Comment:
is all of this really necessary? I imagine this should be fixable by just giving `styles` an explicit type annotation, something like

```ts
const style: { [key: string]: React.CSSProperties } = {
    ...
};
```